client/jetpack-onboarding/style.scss

```diff
@@ -1,4 +1,8 @@
 .jetpack-onboarding {
+	.steps__main {
+		animation: appearWithScaling .3s ease-in-out;
```

Comment:
Nit: a slower rate (e.g. `.8s`) seems easier on the eyes to me. Those blocks are large and fast changes create a flickery effect.

Reply:
Tried that, but I think Michael's suggestion worked really well in mitigating the flickery effect you're talking about (increasing initial scale from 0.5 to 0.9).